src/bindings/python/package/command_line.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+
+import os
+import subprocess
+import sys
+
+
+BIN_DIR = os.path.join(os.path.dirname(__file__), 'bin')
+
+
+def call_program(name, args):
+    return subprocess.call([os.path.join(BIN_DIR, name)] + args, close_fds=False)
```

Comment:
`close_fds` makes me a little bit nervous.

Reply:
If memory serve, this was directly copied from [ninja python wheel](https://github.com/scikit-build/ninja-python-distributions/blob/master/src/ninja/__init__.py#L51), I'm not sure why they used this flag exactly so I removed in for now, thanks.